projects/core/.storybook/manager-head.html

```diff
@@ -26,10 +26,12 @@
       m.parentNode.insertBefore(a, m);
     })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
     ga('create', 'UA-86120402-1', 'auto');
+    ga('set', 'page', new URLSearchParams(window.location.search).get('path'));
     ga('send', 'pageview');
     const pushState = window.history.pushState;
     window.history.pushState = function () {
       pushState.apply(window.history, arguments);
+      ga('set', 'page', new URLSearchParams(window.location.search).get('path'));
       ga('send', 'pageview');
```

Comment:
I think it would good to put these lines in a `sendPageView` function instead of duplicating.